### PR TITLE
Include username into members with payload

### PR DESCRIPTION
**Pull request by xenirio**

### CODE REVIEW

**Comment by martianatwork** [comment]
Hey, I believe username as field name will be better than nickname.

File: app/models/member.rb [hunk 1]
```diff
@@ -191,6 +193,8 @@ def search(field: nil, term: nil)
       case field
       when 'email'
         where("email LIKE ?", term)
+      when 'nickname'
+        where("nickname LIKE ?", term)
```
Comment (calj) [inline app/models/member.rb]: please remove this code, it's not used anywhere.

File: db/migrate/20210120135842_add_username_to_members.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddUsernameToMembers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :members, :username, :string, null: true
```
Comment (calj) [inline db/migrate/20210120135842_add_username_to_members.rb:3]: we need a unique index here

**Review by mnaichuk — APPROVED** [review]

**Comment by calj** [comment]
please PR 2.6

**Pull request closed without merging**